test/app/extra.js

```diff
@@ -1,15 +1,27 @@
 import React from "react"
-import Header from "./.src/extra/header"
-import Body from "./.src/extra/body"
+
+import Root from "./.src/extra/root"
 
 class Box extends React.Component {
+  constructor(props) {
```

Comment:
Yeah, this is a bit of a problem. Because with `react-proxy@1` it would "detect" the change and force the update automatically, but `react-hot-loader@3` doesn't do that anymore (for good reason), but it means stuff like external components are much harder to live update because something has to load the new module and force the re-render. Kind of like what `AppContainer` does. In fact maybe we can just wrap this component in `AppContainer` and that would fix the issue. Agree this isn't perfect but I figured it was okay for a test.